examples/dep/echarts-jsdom/github.json

```diff
@@ -1,7 +1,7 @@
 {
   "owner": "obsidiansystems",
   "repo": "echarts-jsdom",
-  "branch": "master",
-  "rev": "5a3fc689a1ddb52eafcb511b4a05f30099f71c6e",
-  "sha256": "0qwd6kc9g1vf9y5cr3qdbmdxgzjmf51jlgryxjbfs6gcmjp7z6si"
+  "branch": "dn-add-on-events",
```

Comment:
This will need to be updated once https://github.com/obsidiansystems/echarts-jsdom/pull/1 is merged